examples/auth_example/auth_example_server/lib/src/generated/protocol.dart

```diff
@@ -52,14 +64,22 @@ class Protocol extends _i1.SerializationManagerServer {
   String? getClassNameForObject(Object? data) {
     String? className = super.getClassNameForObject(data);
     if (className != null) return className;
-    if (data is _i4.Example) {
+    if (data is _i6.Example) {
       return 'Example';
     }
     className = _i2.Protocol().getClassNameForObject(data);
     if (className != null) {
       return 'serverpod.$className';
     }
     className = _i3.Protocol().getClassNameForObject(data);
+    if (className != null) {
```

Comment:
This is problematic, as the `className` used for serialization would change with the file being moved to the new package (from `serverpod_auth.X` to `serverpod_auth_email.X`).

Thus this would make this a breaking change towards the client.

Also tricky is the fact that since this protocol defers to each package's helper, it wasn't initially obvious, that the server lost the e-mail endpoints, because the protocol was unchanged (and no methods visibly removed).

The good news is, that the database storage survived the move without issue, as no migration was created (supposedly the tables are not prefixed, to be double checked).

Reply:
```suggestion
    if (className != null && !className.contains('.')) {
```

Could be a quick-fix to support fully qualified name (if they want to "pretend" to be from a different module). But the issue would also occur on deserialization, so maybe a better structure to explicitly list these "renames" would make sense, such that the same can also be used for incoming messages.